zerver/openapi/openapi.py

```diff
@@ -291,7 +292,35 @@ def validate_against_openapi_schema(content: Dict[str, Any], path: str,
         # TODO: Remove this after all events have been documented.
         fix_events(content)
     validator = OAS30Validator(schema)
-    validator.validate(content)
+    try:
+        validator.validate(content)
+    except ValidationError as error:
```

Comment:
Do we want this exception handler unconditionally?  It might make sense to just run it for `test_events.py` (by passing an argument controlling it), since I think the errors are reasonable in size for other endpoints?

Reply:
Yes we can control it to run just for `test_events.py`